OpenRA.Mods.Common/Traits/World/Locomotor.cs

```diff
@@ -186,12 +199,9 @@ public int MovementSpeedForCell(CPos cell)
 			return terrainInfos[index].Speed;
 		}
 
-		public short MovementCostToEnterCell(Actor actor, CPos destNode, BlockedByActor check, Actor ignoreActor)
+		public short MovementCostToEnterCell(Actor actor, CPos srcNode, CPos destNode, BlockedByActor check, Actor ignoreActor)
 		{
-			if (!world.Map.Contains(destNode))
```

Comment:
This safety check wasn't ported over.

Reply:
`MovementCostForCell` returns `MovementCostForUnreachableCell` if the first parameter is not contained in the map, so we're safe here.